src/main/java/frc/robot/Config.java

```diff
@@ -0,0 +1,107 @@
+package frc.robot;
+/*
+ * Constant values used throughout robot code.
+ * In "C" this would be a header file, but alas, this is Java
+ */
+// @Deprecated
+// public class Constants {
```

Comment:
Delete commented out code (here and below)

Reply:
Fixed